var/spack/repos/builtin/packages/fj/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import re
+
+import llnl.util.tty as tty
+import spack.compiler
+import spack.util.executable
+
+
+class Fj(Package):
+    """The Fujitsu compiler system is a high performance, production quality
+    code generation tool designed for high performance parallel
+    computing workloads.
+    """
+
+    homepage = "https://www.fujitsu.com/global/Images/openmp-compiler-datasheet-v5.pdf"
```

Comment:
```suggestion
    homepage = "https://www.fujitsu.com/global/us"
```
The pdf file is too old, but description of Fujitsu compiler is not found yet in Fujitsu web page...